pydax/loaders/_base.py

```diff
@@ -40,7 +44,7 @@ def check_path(self, path: Union[_typing.PathLike, Dict[str, str]]) -> None:
 
         :param path: The path of the file to be loaded.
         :raises TypeError: ``path`` is not a path object.
-        :return: No return unless the path is invalid, in which case see TypeError.
+        :return: No return unless the ``path`` is invalid, in which case see ``TypeError``.
```

Comment:
```suggestion
        :return: No return unless the ``path`` is invalid, in which case see :class:`TypeError`.
```

We can set up interlink with the docs of python and pandas: https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html

Reply:
nice I didn't know about that extension, I'll add it